frontend/public/components/utils/branding.ts

```diff
@@ -0,0 +1,43 @@
+import * as okdLogoImg from '../../imgs/okd-logo.svg';
+import * as openshiftLogoImg from '../../imgs/openshift-logo.svg';
+import * as ocpLogoImg from '../../imgs/openshift-platform-logo.svg';
+import * as onlineLogoImg from '../../imgs/openshift-online-logo.svg';
+import * as dedicatedLogoImg from '../../imgs/openshift-dedicated-logo.svg';
+import * as azureLogoImg from '../../imgs/azure-red-hat-openshift-logo.svg';
+
+export const getBrandingDetails = () => {
```

Comment:
I had to extract this function from `masthead` because there was circular dependency between `masthead` -> `masthead-toolbar` -> `about-modal` -> `masthead`